examples/metrics_simple/README.md

```diff
@@ -1,86 +1,108 @@
 # Simple Metrics Example
 
-This example initializes the metrics pipeline with 2 different instrument types.
-Here are more detailed explanations of each part.
+This example initializes the metrics pipeline with 3 different instrument types -
 
-1: Initialize an exporter and a reader. In this case, we initialize an OStream
+- [Counter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#counter)
+- [Histogram](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#histogram)
+- [Asynchronous/Observable Counter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#asynchronous-counter)
+
+ Here are more detailed steps with explanation. Note that the steps 4, 6, and 8
+ are done in Instrumentation library for creating and recording Instruments,
+ and rest of the steps are done in application to configure SDK.
+
+1. Initialize an exporter and a reader. In this case, we initialize an OStream
 Exporter which will print to stdout by default.
-The reader periodically collects metrics from the collector and exports them.
+The reader periodically collects metrics from the Aggregation Store and exports them.
 
-```cpp
-std::unique_ptr<metric_sdk::MetricExporter> exporter{new exportermetrics::OStreamMetricExporter};
-std::unique_ptr<metric_sdk::MetricReader> reader{
-    new metric_sdk::PeriodicExportingMetricReader(std::move(exporter), options)};
-```
+    ```cpp
+    std::unique_ptr<metric_sdk::MetricExporter> exporter{new exportermetrics::OStreamMetricExporter};
```

Comment:
Please add a
`  using namespace exportermetrics = xxx;`
so the reader knows exactly how to use this code.

Reply:
Thanks. Added all the used aliases at the beginning.